dev/run-tests-jenkins

```diff
@@ -31,4 +31,7 @@ if [[ "$PYTHON_VERSION_CHECK" == "True" ]]; then
   exit -1
 fi
 
+export JAVA_HOME=/usr/java/jdk-11.0.1
+export PATH=${JAVA_HOME}/bin:${PATH}
```

Comment:
if it doesn't work, you can even try within `build/mvn`.

Reply:
Seems it works:
![image](https://user-images.githubusercontent.com/5399861/62992807-7f78db00-be87-11e9-9ad9-442ff1de626e.png)

Reply:
@wangyum, can you try this?

Seems we should pass env at

https://github.com/apache/spark/pull/25423/files#diff-3c9f4fccf7d30ce2e8fa86db2ad1fdadR124

```diff
diff --git a/dev/run-tests-jenkins.py b/dev/run-tests-jenkins.py
index 4b91a5fa423..7a73657138f 100755
--- a/dev/run-tests-jenkins.py
+++ b/dev/run-tests-jenkins.py
@@ -121,7 +121,8 @@ def run_tests(tests_timeout):

     test_result_code = subprocess.Popen(['timeout',
                                          tests_timeout,
-                                         os.path.join(SPARK_HOME, 'dev', 'run-tests')]).wait()
+                                         os.path.join(SPARK_HOME, 'dev', 'run-tests')],
+                                        env=dict(os.environ)).wait()
```

Seems like there's something wrong when environment variables are inherited (by default).

Reply:
cc @shaneknapp and @srowen FYI Yuming is trying to build it on the top of https://github.com/apache/spark/pull/25423 . Seems Java 8 is not properly set so it needs some changes at #25423

Reply:
I just pushed two commits to test